modules/sbo-categories-of-exposable-binding-data.adoc

```diff
@@ -129,7 +146,7 @@ metadata:
 
 The following example shows how to expose a resource definition value as OLM descriptors:
 
-.Example: Exposing  a resource definition value as OLM descriptors
+.Example: Exposing a resource definition value as OLM descriptors
```

Comment:
```suggestion
.Example: Exposing a resource definition value using OLM descriptors
```

Reply:
That `as` -> `using` suggesting would to all example titles where `Exposing ... as ...` is.

Reply:
As discussed replaced all the instances with "through" to maintain consistency across the usage of words.